requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
+git+https://github.com/edx/XBlock.git#egg=XBlock
```

Comment:
@nedbat Does a standard / best practice exist for pinning XBlocks to a particular version of the XBlock module?